.travis.yml

```diff
@@ -5,7 +5,7 @@ sudo: false
 env:
   global:
     - CONFIG_URL=https://raw.githubusercontent.com/ros-infrastructure/ros_buildfarm_config/production/index.yaml
-    - CONFIG2_URL=https://raw.githubusercontent.com/ros2/ros_buildfarm_config/ros2/index.yaml
+    - CONFIG2_URL=https://raw.githubusercontent.com/ros2/ros_buildfarm_config/ci/index.yaml
```

Comment:
Reminder to myself to change this after https://github.com/ros2/ros_buildfarm_config/pull/17 is merged.